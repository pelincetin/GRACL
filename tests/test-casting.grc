int main() {
    int i = 5;
    double d = intToDouble(i);
    printi(i);
    print(doubleToString(d));
    print(doubleToString(intToDouble(23)));
    return 0;
}