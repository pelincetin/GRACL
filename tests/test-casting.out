5
5.000000
23.000000